python/pyspark/sql/group.py

```diff
@@ -260,19 +259,19 @@ def wrapped(*cols):
             import pandas as pd
             result = func(pd.concat(cols, axis=1, keys=columns))
             if not isinstance(result, pd.DataFrame):
-                raise TypeError("Return type of the user-defined function should be "
-                                "Pandas.DataFrame, but is {}".format(type(result)))
+                raise TypeError('Return type of the user-defined function should be '
```

Comment:
BTW, I think both `"` and `'` are fine. Let's avoid such changes here unless there is a strong reason for it.

Reply:
Reverted.